json/wireguard.json

```
@@ -30,5 +30,13 @@
         "username": "admin",
         "password": "admin"
     },
-    "notes": []
-}
\ No newline at end of file
+  "notes": [
+    {
+      "text": "Wireguard and WGDashboard are not the same. More info: https://donaldzou.github.io/WGDashboard-Documentation/what-is-wireguard-what-is-wgdashboard.html",
```

```suggestion
      "text": "Wireguard and WGDashboard are not the same. More info: `https://donaldzou.github.io/WGDashboard-Documentation/what-is-wireguard-what-is-wgdashboard.html`",
```
This will make it a copy block